### ISSUE TITLE
Feature Request: Dropdown similar to the background apps dropdown

### ISSUE THREAD

**Issue by AshtakaOOf** [Rayzeq/quick-settings-audio-panel#67]
Gnome shell version: 47

I enjoy using this extension to change audio output of various apps on the fly, but I don't like the clutter and don't want to have it in a separate panel.
I was hoping to have a setting that adds a dropdown similar to the background apps dropdown.

![image](https://github.com/user-attachments/assets/acb293a3-56f6-4815-8a17-45b48abe189a)
The background apps dropdown in question
Gnome is in French in the screenshot

Thanks for taking your time reading this issue. :grin:

**Comment by Rayzeq** [comment]
Something that would look like this ?
![Untitled](https://github.com/user-attachments/assets/ba47fd61-058d-4b5d-81ad-60ef692ade8b)

**Comment by AshtakaOOf** [comment]
Yeah a dropdown just like this for most audio sources. :grinning:

**Comment by Rayzeq** [comment]
This feature was added in version 75. It may take some time to review on extensions.gnome.org because there are many internal changes, however you can still install it manually.

**Comment by AshtakaOOf** [comment]
Love it! Thanks a lot for adding it. <3

**Comment by AshtakaOOf** [comment]
I do have a nitpick, when the dropdown is enabled I cannot change the audio input/output of apps.

**Comment by Rayzeq** [comment]
Yes, it's stated in the preferences, adding it would unfortunately require a lot more internal changes.
I'll reopen the issue as a reminder to add it

**Comment by alvv-z** [comment]
I'm not sure whether this is the right issue, but I have `Applications mixer` and  `Put the sliders in submenu` enabled, but there is no submenu. I am getting a warning that `pactl` is not installed, but that shouldn't affect the submenu, right?

**Comment by AshtakaOOf** [comment]
Yeah the submenu is broken. (ignore the fact that i forgot to report this issue 5 times)

**Comment by alvv-z** [comment]
Alright, good to know! I was trying it out since sometimes I have too many application sliders and when the total panel height exceeds the display height, the whole menu just disappears.

**Comment by Rayzeq** [comment]
The feature now works again in v91
